docroot/modules/custom/mass_admin_pages/mass_admin_pages.module

```diff
@@ -232,3 +232,36 @@ function mass_admin_pages_preprocess_page(&$variables) {
     unset($variables['page']['content']['mass_admin_theme_local_actions']);
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Hide the KPI tab and fields on all new service pages.
+ */
+function mass_admin_pages_form_node_service_page_form_alter(&$form, &$form_state, $form_id) {
+  unset($form['field_service_kpi_choice']);
+  foreach (array_keys($form) as $field_name) {
+    if (strpos($field_name, 'field_kpi_') !== FALSE) {
+      unset($form[$field_name]);
+    }
+  }
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Hide the KPI tab and fields on all service pages except two PFML pages.
+ */
+function mass_admin_pages_form_node_service_page_edit_form_alter(&$form, &$form_state, $form_id) {
+  $pfml_service_family_nids = [414121, 535101];
```

Comment:
@roberttstephens Could we instead store these via the State API?

Reply:
@tom-fleming I'm happy to make the change if you think it's best. What would be the benefit of using state instead of hard coding the nids for now?

These nids are also stored in https://github.com/massgov/openmass/pull/552/files#diff-4747375ceb5366a98cd74f71459192b08c0f3abfb7f0b97bd82126c3631d0629 , the config that determines if the service family analytics dashboard shows. It seems like if the nids did somehow need to change, other code changes would be necessary even if these nids are pulled from the state API.

Reply:
Oh, wow, I see what you mean. It looks like that contributed module was written for this project, too. I wish that it wasn't storing those values in config either 🙃, but I don't really see the point in rewriting the module for this task. So, I'm okay with this.